tools/StaticAnalysis/Exceptions/BreakingChangeIssues.csv

```diff
@@ -121,4 +121,43 @@
 "D:\workspace\powershell-demand\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Compute\Microsoft.Azure.Commands.Compute.dll","Microsoft.Azure.Commands.Compute.Automation.RestartAzureRmVmss","Restart-AzureRmVmss","0","1050","The parameter set 'InvokeByDynamicParameters' for cmdlet 'Restart-AzureRmVmss' has been removed.","Add parameter set 'InvokeByDynamicParameters' back to cmdlet 'Restart-AzureRmVmss'."
 "D:\workspace\powershell-demand\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Compute\Microsoft.Azure.Commands.Compute.dll","Microsoft.Azure.Commands.Compute.Automation.StartAzureRmVmss","Start-AzureRmVmss","0","1050","The parameter set 'InvokeByDynamicParameters' for cmdlet 'Start-AzureRmVmss' has been removed.","Add parameter set 'InvokeByDynamicParameters' back to cmdlet 'Start-AzureRmVmss'."
 "D:\workspace\powershell-demand\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Compute\Microsoft.Azure.Commands.Compute.dll","Microsoft.Azure.Commands.Compute.Automation.UpdateAzureRmVmssInstance","Update-AzureRmVmssInstance","0","1050","The parameter set 'InvokeByDynamicParameters' for cmdlet 'Update-AzureRmVmssInstance' has been removed.","Add parameter set 'InvokeByDynamicParameters' back to cmdlet 'Update-AzureRmVmssInstance'."
-"D:\workspace\powershell-demand\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Compute\Microsoft.Azure.Commands.Compute.dll","Microsoft.Azure.Commands.Compute.Automation.ConvertToAzureRmVMManagedDisk","ConvertTo-AzureRmVMManagedDisk","0","1050","The parameter set 'InvokeByDynamicParameters' for cmdlet 'ConvertTo-AzureRmVMManagedDisk' has been removed.","Add parameter set 'InvokeByDynamicParameters' back to cmdlet 'ConvertTo-AzureRmVMManagedDisk'."
\ No newline at end of file
+"D:\workspace\powershell-demand\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Compute\Microsoft.Azure.Commands.Compute.dll","Microsoft.Azure.Commands.Compute.Automation.ConvertToAzureRmVMManagedDisk","ConvertTo-AzureRmVMManagedDisk","0","1050","The parameter set 'InvokeByDynamicParameters' for cmdlet 'ConvertTo-AzureRmVMManagedDisk' has been removed.","Add parameter set 'InvokeByDynamicParameters' back to cmdlet 'ConvertTo-AzureRmVMManagedDisk'."
```

Comment:
This is not added.  This line was at the end of the file without new line before, so it appears this line is modified.